pkg/build/apiserver/registry/buildlog/rest_test.go

```diff
@@ -199,13 +298,21 @@ func resourceLocationHelper(buildPhase buildv1.BuildPhase, podPhase string, ctx
 	}
 	storage.getSimpleLogsFn = getSimplePodLogsFn
 
+	// artificially waiting for informer's cache synchronization
+	if !cache.WaitForCacheSync(
+		informersCtx.Done(),
```

Comment:
Assuming using provided context:
```suggestion
		ctx.Done(),
```

Reply:
Please consider [this comment](https://github.com/openshift/openshift-apiserver/pull/206#discussion_r630787588).